javascripts/discourse/components/topic-next-button.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="topic-next" >
+  <DButton class="topic-next-button" @action={{action this.goToNextTopic}} @icon="chevron-right" @label={{if this.site.desktopView (theme-prefix "topic_next_label")}} />
```

Comment:
You can drop the `{{action` helper here. Since the function is decorated with `@action` (and is therefore bound to the component instance), you can just pass a reference to it like `@action={{this.goToNextTopic}}`

Reply:
Thanks @davidtaylorhq!